_rules/css-restrict-orientation-b33eff.md

```diff
@@ -37,6 +37,8 @@ The rule applies to any HTML element that is [visible](#visible) and has a CSS [
 
 The target element is neither rotated clockwise nor counter clockwise around the Z-axis at an angle corresponding to 90 degrees relative from the position of the element in `landscape` orientation to the position of the element in `portrait` orientation, and vice versa.
 
+**Note:** Imagine the display of a smartphone with an upwards pointing arrow its center. If we turn the smartphone a quarter turn, that is move from one orientation to the other, we would want the arrow to continue pointing upwards. This is done by rotating it a quarter turn to counter our change in orientation. In effect, the arrow has remained in place and its rotation relative from one orientation to the other is 0 degrees. Now imagine that we rotate the arrow by a quarter turn _only_ when in one orientation and not the other; its rotation relative from one orientation to the other would then be 90 degrees and it would appear stuck, or locked, as we move between orientations. What we've done is effectively counter the smartphones attempt at countering our change in orientation.
```

Comment:
I still find the note a bit confusing. I think it is because there are a lot of personas (devs, user, smartphone) and it is not clear who the pronoun refer to.
```
**Note:** Imagine the display of a smartphone with an upwards pointing arrow its center. 
If we [=user]
turn the smartphone a quarter turn, that is move from one orientation to the other, 
we [=user ultimately, but also sort of =smartphone deciding this since you can have orientation lock on smartphone and then arrow won't turn]
would want the arrow to continue pointing upwards. 
This is done [=by the smartphone, which is why second "we" in previous sentence feels like smartphone]
by rotating it a quarter turn to counter our change in orientation. 
In effect, the arrow has remained in place and its rotation relative from one orientation to the other is 0 degrees. 
Now imagine that we [=dev?]
rotate the arrow by a quarter turn _only_ when in one orientation and not the other; its rotation relative from one orientation to the other would then be 90 degrees and it would appear stuck, or locked, 
as we [=user] move between orientations. 
What we've [=dev]
done is effectively counter the smartphones [=smartphone]
attempt at countering our [=user's]
change in orientation.
```

Not sure how to express it better…

Reply:
I'll make sure everything is in active voice and be more clear about the "whos".